source/docs/articles/sites/migrate/migrating-from-wpengine.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Migrating WordPress Sites from WP Engine to Pantheon
+description: Get all the details you need to know to successfully migrate your site away from WP Engine.
+category:
+  - WordPress
+  - getting-started
+  - developing
+keywords: wordpress, pantheon, wpengine
+---
+
+Manually migrating a site from WP Engine to Pantheon can be tricky, as you cannot use WP Engine’s backup feature to transfer your site. This is because the backup file excludes many of your site's files and directories. See the full list of [exclusions](https://wpengine.com/support/restore/).
+
+The easiest way to migrate your site is to use the [Pantheon Migration Plugin](https://wordpress.org/plugins/bv-pantheon-migration/) created by BlogVault.
+
+## Migrate Using the Pantheon Migration Plugin
+
+1. You can install the plugin by uploading `bv-pantheon-migration` through the WordPress Plugins menu or to the `/wp-content/plugins/` directory.
+2. After you've activated the plugin, click **Pantheon Migration** in the left navigation menu and enter the required information:
+**Email**: This email address will receive status updates on your migration  
+**Destination URL**: Your Pantheon address you are migrating to. For example: http://dev-sitename.pantheon.io  
+**SFTP Server Address**: Enter the host address found in your Pantheon Dashboard under Connection Information.  
+**SFTP Username**: Enter the SFTP username found in your Pantheon Dashboard under Connection Information.  
+**SFTP Password**: This is the password you use to log in to your Pantheon Dashboard.  
+3. Click **Migrate**. The plugin will automatically verify your SFTP credentials and notify you if any issues are found.
+4. After the migration is complete, click the button to see the results of your migration and visit your Pantheon site.
+
+## Manually Migrate from WP Engine
+
+1. Create and download an archive of your site. Archives can be stored in a single file or as three separate files.
+2. Unzip your site's codebase, files, and database.
```

Comment:
I don't think the files come with. Please double check. The next step indicates they must be manually downloaded separately from the archive.